spec/support/deprecated_classes.rb

```diff
@@ -10,7 +10,7 @@ def self.deprecated_classes
 
   def modified_tag_option(key, value, *rest)
     original_result = original_tag_option(key, value, *rest)
-    return original_result unless key == :class
+    return original_result unless [:class, 'class'].include?(key)
```

Comment:
I was wondering why my lint wasn't getting picked up and discovered that `key` was `"class"` as a string... unsure when it is sent as a symbol. Another option would be to force it to a string/symbol to simplify the comparison? I don't feel strongly about this

Other option:
```suggestion
    return original_result unless key.to_s == 'class'
```

or 
```suggestion
    return original_result unless key.to_sym == :class
```

Reply:
It's fine the way it is although I'd prefer the first option.  I'd shy away from the second option because of to_sym DOS attacks (although in this case the key is not user input)

Reply:
Any of the above seems fine to me. I'd been drawn to one of your alternatives vs. what's currently in the branch, though it does kinda obfuscate what other non-string forms we're expecting. Then again, maybe this is an appropriate place for a sledgehammer approach to normalizing the input.

Reply:
I think first one is clearer 👍🏼

Reply:
Re: DOS attacks:

1. starting with Ruby 2.2, symbols get garbage collected [(source)](https://www.sitepoint.com/symbol-gc-ruby-2-2/#:~:text=Starting%20with%20Ruby%202.2%20symbols%20are%20now%20garbage%20collected.&text=Since%20the%20symbols%20we%20create,many%20objects%20that%20it%20crashes.)
2. this monkeypatch only applies in specs, so it's not a viable attack surface

and based on consensus, will do the `.to_s` approach